Libplanet/Blockchain/BlockChain.cs

```diff
@@ -506,20 +540,25 @@ public Transaction<T> GetTransaction(TxId txId)
             FungibleAssetStateCompleter<T> stateCompleter = null
         )
         {
+            if (offset is null && Count < 1)
```

Comment:
Is `offset is null` condition for interface compability? IMO there is no reason to offset is nullable.

Reply:
It was already fixed.  I guess you read the outdated revision.